Sources/SwiftInspectorVisitors/EnumVisitor.swift

```diff
@@ -149,11 +149,3 @@ public final class EnumVisitor: SyntaxVisitor {
   private var enumInfo: EnumInfo?
   private var innerEnums = [EnumInfo]()
 }
-
-public struct EnumInfo: Codable, Equatable {
```

Comment:
this type is moved to below with no changes (unless you are in file-changed view, at which point the comment is moved to a new type)